scripts/validate-icons

```diff
@@ -5,11 +5,11 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
-ICONS_PATH='frontend/icons/src/main/resources'
```

Comment:
While you're here... this is an unexported constant, I'd rather expect it to be in lower_snake_case. UPPER_SNAKE_CASE is conventionally restricted to exported vars